src/greet.test.ts

```diff
@@ -17,7 +15,8 @@ describe("greet", () => {
 	});
 
 	it("logs to the console once when message is provided as an object", () => {
-		logger.mockImplementation(() => undefined);
+		console.log("This demonstrates console-fail-test failing a test. :)");
```

Comment:
This `console.log` should cause a test failure...

Reply:
...and it does! https://github.com/JoshuaKGoldberg/template-typescript-node-package/actions/runs/4837536031/jobs/8621425803?pr=386

```plaintext
 ❯ src/greet.test.ts  (4 tests | 1 failed) 13ms
   ❯ src/greet.test.ts > greet > logs to the console once when message is provided as an object
     → Oh no! Your test called the following console method:
  * log (1 call)
    > Call 0: "This demonstrates console-fail-test failing a test. :)"
```